src/web/mjs/connectors/MangaHost.mjs

```diff
@@ -49,12 +49,12 @@ export default class MangaHost extends Connector {
     }
 
     _getChapterList( manga, callback ) {
-        this.fetchDOM( this.url + manga.id, 'ul.list_chapters li a[data-original-title]' )
+        this.fetchDOM( this.url + manga.id, 'div.chapters div.card.pop div.pop-title span' )
             .then( data => {
                 let chapterList = data.map( element => {
                     return {
-                        id: [manga.id, element.id].join('/'),
-                        title: element.dataset.originalTitle.replace( manga.title, '' ).replace( /\s+-\s*$/, '' ).trim(),
+                        id: [manga.id, element.textContent].join('/'),
```

Comment:
Looks weird that the ID is the title (USVString), but i guess it has been tested with various chapter titles and works...

Reply:
There is no chapter title available on the page :\
https://mangahost2.com/manga/hack-g-u-mh35138
the only i was able to find is the span that is mostly the chapter number, but sometimes EXTRA-01